docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -237,14 +273,74 @@ class DebugPage extends React.PureComponent<Props, State> {
 
   private onCopy = () => {
     const response = this.state.debugResponse;
-    const textArea = document.createElement('textarea');
-    textArea.style.opacity = '0.0';
-    textArea.value = response;
-    document.body.appendChild(textArea);
-    textArea.focus();
-    textArea.select();
-    document.execCommand('copy');
-    document.body.removeChild(textArea);
+    if (response.length > 0) {
```

Comment:
An empty string cannot be copied to the clipboard, at least on my browser.

Reply:
Or disable the button?

Reply:
I like @trustin's idea. 😆

Reply:
Yup. 😆